tests/operator/operator.go

```diff
@@ -765,10 +765,10 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 		}
 
 		startAllVMIs = func(vmis []*v1.VirtualMachineInstance) {
-			for _, vmi := range vmis {
-				vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+			for i := range vmis {
+				vmis[i], err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmis[i])).Create(context.Background(), vmis[i], metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred(), "Create VMI successfully")
-				libwait.WaitForSuccessfulVMIStart(vmi)
+				libwait.WaitForSuccessfulVMIStart(vmis[i])
```

Comment:
If you are at it, please also update the vmi status after the waiting.
`vmis[i] = WaitForSuccessfulVMIStart(vmi[i])`